integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -460,3 +459,15 @@ def test_debug_get_raw_transaction_non_existent_tx_hash(self):
             response["error"]["message"]
             == "Empty Neon transaction receipt for 0xd9765b77e470204ae5edb1a796ab92ecb0e20fea50aeb09275aea740af7bbc69"
         )
+
+    def test_struct_log_and_call_tracer_to_precompiled_contract(self, pytestconfig, evm_loader):
+        address = "0x0000000000000000000000000000000000000006"
+        sender_account = self.accounts[0]
+        amount = random.randint(1, 10)
+        instruction_tx = self.web3_client.make_raw_tx(sender_account.address, address, amount=amount, estimate_gas=True)
```

Comment:
looks like an empty transaction
you don't call precompile contract here
just send some amount (it's better to delete amount at all)